code/go/0chain.net/miner/chain_test.go

```diff
@@ -0,0 +1,390 @@
+package miner
+
+import (
+	"0chain.net/chaincore/block"
+	"0chain.net/chaincore/chain"
+	"0chain.net/chaincore/client"
+	round_mocks "0chain.net/chaincore/mocks"
+	"0chain.net/chaincore/node"
+	"0chain.net/chaincore/round"
+	"0chain.net/chaincore/state"
+	"0chain.net/chaincore/threshold/bls"
+	"0chain.net/core/common"
+	"0chain.net/core/datastore"
+	"0chain.net/core/logging"
+	"0chain.net/core/memorystore"
+	mocks "0chain.net/core/mocks"
+	"context"
+	"github.com/gomodule/redigo/redis"
+	"github.com/rafaeljusto/redigomock"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/suite"
+	"go.uber.org/zap"
+	"net/http"
+	"testing"
+)
+
+type ChainTestSuite struct {
+	suite.Suite
+}
+
+func TestChainTestSuite(t *testing.T) {
+	logging.Logger = zap.NewNop()
+
+	//round.SetupEntity()
+
+	suite.Run(t, &ChainTestSuite{})
+}
+
+func (s *ChainTestSuite) TestChainStarted() {
+	c := chain.NewChainFromConfig()
+	SetupMinerChain(c)
+
+	mc := GetMinerChain()
+
+	mc.ChainStarted(context.Background())
+
+}
+
+func (s *ChainTestSuite) TestCreateRound() {
```

Comment:
You can skip doing tests for simple functions like accessors or object creation.